storybook/src/example-pages/toepassingen/loket/homepage.ts

```diff
@@ -0,0 +1,143 @@
+import { html } from "lit-html";
+
+import { examplePageFactory } from "../../../example-page-factory";
+import { footerPartial } from "../../partials/footer";
+import { headerPartial } from "../../partials/header";
+import { header } from "../../partials/header.content";
+import { linkList } from "./homepage.content";
+
+examplePageFactory(
+  "Toepassingen/Loket",
+  "Homepage",
+  (
+    { anchorTemplate, linkListTemplate, highlightBoxTemplate, richContentTemplate, tileTemplate, buttonTemplate },
+    templates
+  ) => html`
+    <div class="container">
+      ${headerPartial(templates, header)}
+      <main>
+        <div
+          class="row dso-banner no-button-banner dso-banner-implementation-specific-image"
+          style="background-image: url('images/hero2.jpeg')"
+        >
+          <div class="col-lg-6 col-sm-8">
+            ${highlightBoxTemplate({
+              white: true,
+              content: richContentTemplate({
+                children: html`
+                  <h1>Homepage</h1>
+                  <p>
+                    Een vergunning aanvragen of melding doen, bijvoorbeeld voor een nieuwe dakkapel, een nieuw
+                    bedrijfspand of een activiteit op of aan een dijk. Het kan met het nieuwe Omgevingsloket.
+                  </p>
+                `,
+              }),
+            })}
+          </div>
+        </div>
+        <div class="row dso-featured dso-equal-heights">
+          <div class="col-xs-12">
+            ${highlightBoxTemplate({
+              white: true,
+              border: true,
+              content: richContentTemplate({
+                children: html`
+                  <div class="row">
+                    <div class="col-md-5 col-md-push-7">
+                      <div class="dso-rich-content">
+                        <h2>Check of u een vergunning nodig heeft</h2>
+                        <p>Wilt u iets veranderen aan het huis, tuin of bedrijf</p>
+                        <p>
+                          Doe hier de vergunningcheck om te kijken of u een vergunning nodig heeft of een melding moet
+                          doen.
+                        </p>
+                        ${anchorTemplate({ label: "Vergunningcheck", url: "#", modifier: "dso-secondary" })}
+                      </div>
+                    </div>
+                    <div class="col-md-7 col-md-pull-5">
+                      <div class="dso-rich-content">
+                        <h3>Meest gekozen werkzaamheden</h3>
+                      </div>
+                      <div class="dso-tile-grid">
+                        ${tileTemplate({ anchor: "#", image: "#", label: "boom kappen", variant: "theme" })}
```

Comment:
Nu komen er 'dode' plaatjes in de markup (met een lege `src`).

Reply:
Dat is het punt wat ik eerdere maakte. Ik heb geen flauw idee waar ik een afbeelding/illustratie kan aanroepen. 

Weet jij wat ik hier bij src kan invullen?